pyro/infer/inspect.py

```diff
@@ -604,9 +604,8 @@ def render_model(
     graph = render_graph(graph_spec, render_distributions=render_distributions)
 
     if filename is not None:
-        filename = Path(filename)
-        suffix = filename.suffix[1:]  # remove leading period from suffix
-        graph.render(filename.stem, view=False, cleanup=True, format=suffix)
+        suffix = Path(filename).suffix[1:]  # remove leading period from suffix
+        graph.render(os.path.splitext(filename)[0], view=False, cleanup=True, format=suffix)
```

Comment:
it seems that you need to import os

Reply:
Ok, I re-added it. Not sure why  it was not there

Reply:
could you run `make format` to fix the lint issue?

Reply:
@fehiepsi  Ops, sorry I am a little beginner. I am not sure what `make format` means here. Is it another package?

I did run **pylint** over the script with this output: 
```
 ************* Module pyro.infer.inspect
inspect.py:1:0: C0114: Missing module docstring (missing-module-docstring)
inspect.py:15:0: R0402: Use 'from pyro import poutine' instead (consider-using-from-import)
inspect.py:26:0: C0116: Missing function or method docstring (missing-function-docstring)
inspect.py:37:4: C0103: Variable name "fn" doesn't conform to snake_case naming style (invalid-name)
inspect.py:39:8: C0103: Variable name "fn" doesn't conform to snake_case naming style (invalid-name)
inspect.py:46:0: C0115: Missing class docstring (missing-class-docstring)
inspect.py:61:0: R0914: Too many local variables (24/15) (too-many-locals)
inspect.py:183:20: E1102: poutine.trace is not callable (not-callable)
inspect.py:204:12: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
inspect.py:206:16: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
inspect.py:211:8: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
inspect.py:212:12: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
inspect.py:212:15: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
inspect.py:221:8: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
inspect.py:223:12: C0103: Variable name "u1" doesn't conform to snake_case naming style (invalid-name)
inspect.py:223:16: C0103: Variable name "p1" doesn't conform to snake_case naming style (invalid-name)
inspect.py:224:16: C0103: Variable name "u2" doesn't conform to snake_case naming style (invalid-name)
inspect.py:224:20: C0103: Variable name "p2" doesn't conform to snake_case naming style (invalid-name)
inspect.py:61:0: R0912: Too many branches (13/12) (too-many-branches)
inspect.py:285:20: E1102: poutine.trace is not callable (not-callable)
inspect.py:323:12: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
inspect.py:323:15: C0103: Variable name "pv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:324:12: C0103: Variable name "pv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:325:16: C0103: Variable name "q" doesn't conform to snake_case naming style (invalid-name)
inspect.py:325:19: C0103: Variable name "qv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:326:16: C0103: Variable name "qv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:323:12: W0612: Unused variable 'p' (unused-variable)
inspect.py:350:19: C0103: Argument name "fn" doesn't conform to snake_case naming style (invalid-name)
inspect.py:358:0: R0914: Too many local variables (19/15) (too-many-locals)
inspect.py:375:12: C0103: Variable name "rv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:382:8: C0103: Variable name "rv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:358:0: R0912: Too many branches (17/12) (too-many-branches)
inspect.py:449:21: W0612: Unused variable 'value' (unused-variable)
inspect.py:466:0: R0914: Too many local variables (27/15) (too-many-locals)
inspect.py:475:8: W0621: Redefining name 'graphviz' from outer scope (line 21) (redefined-outer-name)
inspect.py:475:8: C0415: Import outside toplevel (graphviz) (import-outside-toplevel)
inspect.py:476:4: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
inspect.py:507:12: C0103: Variable name "rv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:542:12: C0103: Variable name "rv" doesn't conform to snake_case naming style (invalid-name)
inspect.py:466:0: R0912: Too many branches (16/12) (too-many-branches)
inspect.py:525:16: W0612: Unused variable 'plate2' (unused-variable)
inspect.py:556:0: R0913: Too many arguments (6/5) (too-many-arguments)

-----------------------------------
Your code has been rated at 8.00/10
```
It does not point anything at lines 607-610, so I assumed it was ok ...

Reply:
I think you can go to the cloned numpyro folder and run `make format`. If you are using windows, probably running `black *.py pyro` and `isort .` will work. You need to install both `black` and `isort` to run those commands (`pip install black isort`)

Reply:
I guess the line is too long, you can do
```
filepath = os.path.splitext(filename)[0]
graph.render(filepath, ...)

Reply:
Okey , dok, thanks for the patience, done!

Reply:
Thanks, CI passed. It seems that you are using a black version different from CI.

Reply:
Perhaps, I am using:
```

black, 23.1.0 (compiled: yes)
Python (CPython) 3.8.13
```